lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/principal/PrincipalPrefixDAOImpl.java

```diff
@@ -234,6 +286,28 @@ public void clearPrincipal(Long principalId) {
 				Long.class, teamId, processed + WILDCARD, limit, offset);
 	}
 
+	@Override
+	public List<Long> listCertainTeamMembersForPrefix(String prefix, Long teamId,
```

Comment:
explicit list for inclusions/exclusions instead of boolean determining behavior

Reply:
you will need to build up a sql query based on the list.

also document exclusions will take precedence over inclusions